mmv1/third_party/terraform/services/compute/resource_compute_disk_test.go.tmpl

```diff
@@ -1644,6 +1668,187 @@ resource "google_compute_disk" "foobar" {
 `, diskName, storagePoolUrl)
 }
 
+func TestExpandStoragePoolUrl_withDataProjectAndZone(t *testing.T) {
+	config := &transport_tpg.Config{
+		ComputeBasePath: "https://www.googleapis.com/compute/v1/",
```

Comment:
It would be great to have values set here so that we can verify that the data values take precedence.

```suggestion
		ComputeBasePath: "https://www.googleapis.com/compute/v1/",
		Project: "other-project",
		Zone: "other-zone",
```